server/src/HttpServer.hs

```diff
@@ -60,6 +64,8 @@ postModification :: (Scotty.ScottyError e, MonadIO m) => Core -> Store.Modificat
 postModification core op = do
   -- the parameter is parsed as type (), therefore only presence or absence is important
   durable <- maybeParam "durable"
+  crash <- maybeParam "crash"
+  if crash == Just () then Scotty.liftAndCatchIO (error "Error ") else pure ()
```

Comment:
This looks like a leftover from debugging, I don't think we need or want this in production.